.buildkite/scripts/e2e-pipeline/bootstrap.py

```diff
@@ -5,17 +5,27 @@
     - Clone integrations repo and prepare packages
     - When E2E finishes, teardown the stack
 """
+import glob
 import os
 import sys
 import time
 import util
+import ruamel.yaml
 
+YAML = ruamel.yaml.YAML()
 
 class Bootstrap:
     ELASTIC_PACKAGE_DISTRO_URL = "https://api.github.com/repos/elastic/elastic-package/releases/latest"
     LOGSTASH_CONTAINER_NAME = "elastic-package-stack-e2e-logstash-1"
     PLUGIN_NAME = "logstash-filter-elastic_integration"
 
+    SUPPORTED_PROCESSORS: list = [
```

Comment:
it would be nice to codify this in a shared yaml that is consumed by [the EventProcessorBuilder](https://github.com/elastic/logstash-filter-elastic_integration/blob/63aa130ba9c76a97bf317c124b5c1f0a5ef1f50e/src/main/java/co/elastic/logstash/filters/elasticintegration/EventProcessorBuilder.java#L94-L131).

Not a blocker, though. Could you file an issue to unify it?

Reply:
This is good idea! Otherwise we need to maintain both places. Yup let me see if I can do it with change. Otherwise I will create an issue and follow up later.

Reply:
Created an issue: https://github.com/elastic/logstash-filter-elastic_integration/issues/249
We can utilize https://github.com/c2nes/javalang to parse the Java class and compare with the processors found from the file. I will follow up and add separately.